Java_String_Reverse_17.java

```diff
@@ -0,0 +1,12 @@
+import java.util.*;
+
+public class Java_String_Reverse_17 {
+    public static void main(String[] args) {
+        //https://www.hackerrank.com/challenges/java-string-reverse/problem?isFullScreen=true
+        
+        Scanner sc=new Scanner(System.in);
+        String A=sc.next();
```

Comment:
Do not use random string as variable name. Do follow proper naming convention.

**Resource for naming variable:**
https://www.theserverside.com/feature/Java-naming-conventions-explained
https://www.geeksforgeeks.org/rules-for-variable-declaration-in-java/